Keyboard navigation | Focus on selection

phylor opened the pull request:
In long lists one would probably set a `max-height` and `overflow-y: scroll`, so that the result list does not grow endlessly.

In that scenario, the current behavior allows keyboard navigation, but the selected elements are not moved into the view (you don't see what you are selecting). This PR suggests to scroll the currently selected element into the view.

The result looks like this:

![screengrab-20210118-1516](https://user-images.githubusercontent.com/1394828/104926671-c1714f00-59a0-11eb-93e3-2404acdffb09.gif)

drnic commented:
@phylor I'm sorry I didn't review this PR earlier; yes, it would be awesome to have the selected item scroll into view.

Since part of the solution is in the example HTML, could I ask that you write a tiny section in the README docs too about suggested HTML?

phylor commented:
@drnic Sure, added a comment! I guess there is no need to explicitly talk about the keyboard navigation, right?

I also deleted my commit changing the examples as you moved them out of the repo.

drnic commented:
Thanks @phylor
